opencensus.go

```diff
@@ -85,6 +86,58 @@ func (c composableRegister) Register(ctx context.Context, cfg Config, vs []*view
 	c.setDefaultSampler(cfg.SampleRate)
 	c.setReportingPeriod(time.Duration(cfg.ReportingPeriod) * time.Second)
 
+	// modify metric tags
+	// ref: https://godoc.org/go.opencensus.io/plugin/ochttp#pkg-variables
+	if cfg.Exporters.Prometheus != nil {
+		for _, view := range vs {
+			// client metrics (method + statuscode tags are enabled by default)
+			if strings.Contains(view.Name, "http/client") {
+				// Host
+				if cfg.Exporters.Prometheus.HostTag {
+					view.TagKeys = appendIfMissing(view.TagKeys, ochttp.KeyClientHost)
+				}
+
+				// Path
+				if cfg.Exporters.Prometheus.PathTag {
+					view.TagKeys = appendIfMissing(view.TagKeys, ochttp.KeyClientPath)
+				}
+
+				// Method
+				if cfg.Exporters.Prometheus.MethodTag {
+					view.TagKeys = appendIfMissing(view.TagKeys, ochttp.KeyClientMethod)
+				}
+
+				// StatusCode
+				if cfg.Exporters.Prometheus.StatusCodeTag {
+					view.TagKeys = appendIfMissing(view.TagKeys, ochttp.KeyClientPath)
```

Comment:
the key and the value doesn't match (status code vs path)

Reply:
Thanks, didnt notice this one when i testet since the status code is present almost everywhere by default.